### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by BeepBot99** [jgdevlabs/aspnetcore-recaptcha#24] [Ideas]
I could help write documentation if needed. Great package!

**Comment by jooni91** [comment]
Thank you for the kind words! I'd really appreciate your help with the documentation. You're welcome to dive in, and if you have any questions, I'll try to assist as best as I can.

**Reply by BeepBot99** [reply]
Is there already any documentation? Also, where and what format?

**Reply by jooni91** [reply]
The only project-level (setup and usage) documentation we currently have is in the README at the root of the project. Additionally, the code XML documentation can be found https://github.com/griesoft/aspnetcore-recaptcha/tree/master/docs. I don't have a specific format in mind, so if you have any ideas or preferences, please feel free to share. Using the wiki in the GitHub repo sounds like a great option to me. What do you think?

**Reply by BeepBot99** [reply]
Sure, wiki sounds good. I believe you have to enable it first though.
